ocaml/src/tools/alba_partial_read_stubs.c

```diff
@@ -97,10 +99,24 @@ static void worker_partial_read(struct job_partial_read* job)
         do {
             int sent = sendfile(job -> socket, in_fd, &offset, todo);
             if(sent == -1){
-                job -> result = -1;
-                job -> errno_copy = errno;
-                close(in_fd);
-                return;
+                int ok = -1;
+                if (errno == EAGAIN || errno == EWOULDBLOCK){
```

Comment:
seems like it is indeed not mentioned in the man page, OTOH I guess it doesn't hurt